<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,10,1">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QPushButton" name="open_folder">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Open Folder</string>
       </property>
      </widget>
      <widget class="QListWidget" name="qlist_images">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>47</y>
         <width>81</width>
         <height>531</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="layoutMode">
        <enum>QListView::Batched</enum>
       </property>
       <property name="batchSize">
        <number>20</number>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="qlabel_image">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="ZoomIn">
         <property name="text">
          <string>Zoom In</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ZoomOut">
         <property name="text">
          <string>Zoom Out</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toggle_move">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Move Image</string>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Move</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../Downloads/icons/move.png</normaloff>../../../Downloads/icons/move.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file </string>
    </property>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionzoom">
   <property name="text">
    <string>zoom </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>qlist_images</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>item_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_folder</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>item_click()</slot>
  <slot>selectDir()</slot>
 </slots>
</ui>
